README.md

```diff
@@ -6,7 +6,12 @@ You can get a variety of different ESP8266 boards here in HCM, for less than $3.
 
 ## [Arduino C projects](arduino_c)
 
-### OLED Display driver sh1106 and ssd1306
+### [OLED Display driver sh1106 and ssd1306](./arduino_c/oled_animation_demo_sh1106)
+
+After some search I found the great libraries [u8glib](https://github.com/olikraus/u8glib) and [u8g2](https://github.com/olikraus/u8g2) from [Oli Kraus](https://github.com/olikraus). Just uncomment the correct line, connnect the 4 needed wires for the I2C connection of the display and you get the first output. The main libraries are 
+
+- [u8glib](https://github.com/olikraus/u8glib) no longer active developed
+- [u8g2](https://github.com/olikraus/u8g2) for monochrome displays, includes __U8x8__ as well
```

Comment:
No special driver included, just u8g2 and g8x8 from Oli Kraus. Reference is great.